include/parser/independent/expression-compiler-clike.hpp

```diff
@@ -41,202 +50,246 @@ namespace CLikeExpressionCompilers {
-                                        40,
-                                        [](const IntType& f,
-                                           const IntType& s,
-                                           IntType& out,
-                                           CompileErrorList& errors) -> bool {
-                                          out = f + s;
-                                          return true;
-                                        }},
-      ExpressionBinaryOperator<IntType>{ExpressionOperatorAssociativity::LEFT,
-                                        "-",
-                                        40,
-                                        [](const IntType& f,
-                                           const IntType& s,
-                                           IntType& out,
-                                           CompileErrorList& errors) -> bool {
-                                          out = f - s;
-                                          return true;
-                                        }},
-      ExpressionBinaryOperator<IntType>{ExpressionOperatorAssociativity::LEFT,
-                                        "*",
-                                        30,
-                                        [](const IntType& f,
-                                           const IntType& s,
-                                           IntType& out,
-                                           CompileErrorList& errors) -> bool {
-                                          out = f * s;
-                                          return true;
-                                        }},
-      ExpressionBinaryOperator<IntType>{ExpressionOperatorAssociativity::LEFT,
-                                        "/",
-                                        30,
-                                        [](const IntType& f,
-                                           const IntType& s,
-                                           IntType& out,
-                                           CompileErrorList& errors) -> bool {
-                                          out = f / s;
-                                          return true;
-                                        }},
-      ExpressionBinaryOperator<IntType>{ExpressionOperatorAssociativity::LEFT,
-                                        "%",
-                                        30,
-                                        [](const IntType& f,
-                                           const IntType& s,
-                                           IntType& out,
-                                           CompileErrorList& errors) -> bool {
-                                          out = f % s;
-                                          return true;
-                                        }}};
+      ExpressionBinaryOperator<IntType>{
+          ExpressionOperatorAssociativity::LEFT,
+          "||",
+          120,
+          [](const IntType& f,
```

Comment:
Please use better variable names, I have no idea what `f` or `s` are from reading this